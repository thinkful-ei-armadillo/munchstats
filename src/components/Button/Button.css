button{
  font-family: "Ubuntu", sans-serif;
}