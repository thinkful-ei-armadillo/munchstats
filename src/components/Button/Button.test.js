import React from 'react';
import Button from './Button';
import {MemoryRouter} from 'react-router-dom';
import {mount} from 'enzyme';
import renderer from 'react-test-renderer';

describe('<Button />', () => {
  it('Renders without crashing', () => {
    mount(<MemoryRouter><Button /></MemoryRouter>);
  });
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MemoryRouter><Button/></MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});