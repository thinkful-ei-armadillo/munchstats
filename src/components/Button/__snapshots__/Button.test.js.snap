// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders the UI as expected 1`] = `
<button
  className="Button"
/>
`;
